README.md

```diff
@@ -1,3 +1,4 @@
 # hello-world
+Just practicing
```

Comment:
In markdown you should add a blank line after a heading

Reply:
Ok, noted!